database/seed.js

```diff
@@ -0,0 +1,85 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+
+mongoose.connect('mongodb://localhost/metadata', {useNewUrlParser: true, useUnifiedTopology: true});
+
+//create our metadata Schema
+const metaSchema = new mongoose.Schema({
+    gameId: Number,
+    singlePlayer: Boolean,
+    onlineCoop: Boolean,
+    lanCoop: Boolean,
+    steamCloud: Boolean,
+    EULAnotice: Boolean,
+    languages: Array,
+    rating: String,
+    gameTitle: String,
+    gameGenre: String,
+    developer: String,
+    publisher: String,
+    franchise: String,
+    releaseDate: Date
```

Comment:
It looks like there is one property that differs ('earlyAccessDate'), so if it is a different schema, I would recommend changing the name, so they aren't both 'metaSchema'; otherwise, I would only build one and export it.